.github/workflows/dart.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 jobs:
   job_001:
-    name: "OS: linux; SDK: dev; PKG: mono_repo; TASKS: `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
+    name: "linux; Dart dev; PKG: mono_repo; `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
```

Comment:
In the name of shortening though, I think we could actually drop the `Dart` portion here? so it would start with just `linux; dev;` for instance.

Reply:
"2.10.0" without context could be confusing – maybe?

Are we talking about the linux version? the package version?

Thoughts?

Reply:
I don't think it would be super confusing, especially once you understand the tool and its output, but I could see the argument for leaving it.